Fix the path to include files
diff --git a/Examples/C++/radiolabelling.cpp b/Examples/C++/radiolabelling.cpp
index cea4917..ca1ff7e 100644
--- a/Examples/C++/radiolabelling.cpp
+++ b/Examples/C++/radiolabelling.cpp
@@ -1,6 +1,6 @@
 #include <iostream>
 #include <cassert>
-#include "../../IsoSpec++/fixedEnvelopes.h"
+#include <IsoSpec++/fixedEnvelopes.h>
 
 using namespace IsoSpec;
 
diff --git a/Examples/C++/water.cpp b/Examples/C++/water.cpp
index 94580dc..56dc2c7 100644
--- a/Examples/C++/water.cpp
+++ b/Examples/C++/water.cpp
@@ -1,7 +1,7 @@
 #include <iostream>
 #include <cassert>
-#include "../../IsoSpec++/isoSpec++.h"
-#include "../../IsoSpec++/fixedEnvelopes.h"
+#include <IsoSpec++/isoSpec++.h>
+#include <IsoSpec++/fixedEnvelopes.h>
 
 using namespace IsoSpec;
 
