diff --git a/CMakeLists.txt b/CMakeLists.txt
index d839ba8..3fa336b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -15,7 +15,7 @@ cmake_minimum_required(VERSION 3.12)
 ############################################################
 # Basic information about project
 
-project(IsoSpec VERSION 2.3.0
+project(IsoSpec VERSION 2.3.0
     DESCRIPTION "A program to calculate isotopic clusters"
     HOMEPAGE_URL "https://github.com/MatteoLacki/IsoSpec")
 
